modules/ROOT/pages/_partials/task-view-governed-apis-for-rulesets.adoc

```diff
@@ -0,0 +1,27 @@
+// Used in docs-exchange/asset-details.adoc and docs-api-governance-documentation/view-conformance-status-in-exchange.adoc
+
+In a ruleset's *Governed APIs* page, you can view the list of centrally-governed APIs to which the ruleset applies. 
+
+Developers can use the *Governed APIs* list to help them improve the quality of the rulesets and APIs they develop as follows: 
+
+* When their APIs fail a ruleset, they can see which other APIs are successfully conforming to the ruleset so that they can fix their APIs.
+* When they are discovering available rulesets to use for their API, they can see which APIs are conformant to a specific ruleset to evaluate how helpful it is.
+
+API consumers can use the *Governed APIs* list when they need to create an API service or application that is conformant to a certain set of standards or rules. They can:
```

Comment:
```suggestion
API consumers can use the *Governed APIs* list when they create an API service or application that conforms to a certain set of standards or rules. They can:
```

Reply:
Same as above in regard to conforms vs. is conformant to. This also needs to imply something other than the edit would achieve.